core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/saved-workflow-section.component.html

```diff
@@ -225,6 +235,12 @@ <h2 class="page-title">Workflows</h2>
         >
           <nz-list-item-meta>
             <nz-list-item-meta-avatar>
+              <input
```

Comment:
I see in my frontend the checkboxes' locations are not very good-looking. They are too close to the workflow number. Try editing the CSS to make it look better.

Reply:
Done.